src/test/java/org/jenkinsci/plugins/ghprb/GhprbRepositoryTest.java

```diff
@@ -109,6 +109,11 @@ public void setUp() throws Exception {
         doReturn(gt).when(trigger).getGitHub();
         
         given(gt.getRepository(anyString())).willReturn(ghRepository);
+
+        // Set rate limit before starting trigger
+        GHRateLimit initRateLimit = new GHRateLimit();
```

Comment:
I'm not clear on how that's the line that needs to be fixed.  The line causing the error is calling `trigger.getGitHub().getRateLimit().remaining` and failing on the remaining property of the rate limit because the mock `GHRateLimit` is not yet associated with the mock `GitHub` class.

I can move the following section before the call to `GhprbTrigger.start`, but then I'd have to update the `verifyGetGithub(callsCount, repoTimes)` call in all the tests that use that method to account for the fact that `getRateLimit` is being called in the `setUp` (indirectly).

    // Mock rate limit
    given(gt.getRateLimit()).willReturn(ghRateLimit);
    increaseRateLimitToDefaults();
    addSimpleStatus();

Let me know your preference and I'll make the associated changes.  Thanks.